GoogleTestAdapter/Tests.Common/ResultChecker/TrxResultChecker.cs

```diff
@@ -61,7 +61,8 @@ public void RunTestsAndCheckOutput(string typeName, string arguments, string tes
                 Assert.Inconclusive($"First time this test runs, created golden file - check for correctness! File: {goldenFile}");
             }
             CheckIfFileIsParsable(goldenFile);
-            if (!CompareXmlFiles(goldenFile, transformedResultFile, out string diffFile))
+            string diffFile;
```

Comment:
Ah I suppose it is for building in VS2015. Based on what Christian said on the email thread, I suppose it is OK not to apply similar changes elsewhere since you are going to switch to VS2017?